invenio_app_ils/permissions.py

```diff
@@ -100,34 +100,25 @@ def loan_extend_circulation_permission(loan):
         document_rec = Document.get_record_by_pid(loan["document_pid"])
         document = document_rec.replace_refs()
         is_overbooked = document.get("circulation", {}).get("overbooked")
-        if is_overbooked == None:
+        if is_overbooked is None:
```

Comment:
Do we have to explicitly check for `None`?

Reply:
yes, read the comment just below, if you remember you developed it. It is an extra check just in case is not `True` or `False` for some reasons.